src/main/java/seedu/duke/template/LrTemplate.java

```diff
@@ -0,0 +1,27 @@
+package seedu.duke.template;
+
+import seedu.duke.component.Inductor;
+import seedu.duke.component.Resistor;
+import seedu.duke.component.VoltageSource;
+
+public class LrTemplate extends Template {
```

Comment:
Later when the `RTemplate` is created this could instead inherit from that.

Reply:
Yup! Good idea